### ISSUE TITLE
"Lic Pro Controller" Connectivity Issue

### ISSUE THREAD

**Issue by Robberino** [CTCaer/jc_toolkit#27]
When trying to change the color of my new PowerA Gamecube Wireless Pro Controller, I can't pass the initial prompt that says that no controllers are connected. The controller is connected via bluetooth, but the program doesn't seem to detect it. I tried to run the program with a regular pro controller, then disconnect it, then connect the Lic Pro Controller, but the program does not detect it. 
Also, instead of a solid led light on the pro controller staying on, the lights continually flash as if it were searching for a device, but it still shows up in my Bluetooth settings as connected.

**Comment by JustColourMePink** [comment]
Same for me. just got a gold Gamecube Wireless Pro Controller, but it won't work with the program for some reason, even though it shows up as a pro controller on the switch.

**Comment by CTCaer** [comment]
I suppose they use a different Vid and Pid than nintendo's.
https://github.com/CTCaer/jc_toolkit/blob/master/jctool/jctool.cpp#L2929

I may add support sometime

**Comment by DrkRussian** [comment]
Yeah, I'm having the exact same issue. Gold PowerA Gamecube inspired one.

**Comment by Olafaloofian** [comment]
I would love to be able to modify the Power A Pro controller colors. How did you find the HID information for the Nintendo controllers? If I can find the codes for these Power A ones I'd be willing to test the program out on mine and submit a pull request if it works. 😊

**Comment by aestellic** [comment]
Same problem here. Has a fix or patch been released for this yet?

**Comment by linkoid** [comment]
Here is the information for the "Lic Pro Controller"
Unfortunately, there doesn't seem to be a strong way to detect it.
![Annotation 2020-12-29 165618](https://user-images.githubusercontent.com/36754150/103319365-9c239d80-49ff-11eb-8dc8-b245b28dd505.png)

**Comment by linkoid** [comment]
I was able to get the controller to connect but if you were hoping to change the colors, the Switch doesn't seem to detect it. If you wanted it for other things though, check pull request #51.

**Comment by AntyXG** [comment]
@linkoid how do i use the pseudo third party controller support? It's just a bunch of code. How does that help me? Please tell me how to use it.

**Comment by MisterGameBoy** [comment]
I restarted my PC and took the batteries out then put the batteries back in and it worked.
